bazel/dep/deps_extra.bzl

```diff
@@ -0,0 +1,22 @@
+#  Copyright 2020 Google LLC
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#       http:# www.apache.org/licenses/LICENSE-2.0
```

Comment:
Ditto. Is this a bug in `addlicense` or bad copy & paste?

Reply:
Fixed. This is just a bad copy paste.